src/test/regress/sql/pg16.sql

```diff
@@ -650,10 +638,142 @@ RESET citus.enable_ddl_propagation;
 -- by default, admin option is false, inherit is true, set is true
 GRANT role3 TO role4;
 GRANT role3 TO role5 WITH ADMIN TRUE, INHERIT FALSE, SET FALSE;
-SELECT roleid::regrole::text AS role, member::regrole::text, admin_option, inherit_option, set_option FROM pg_auth_members WHERE roleid::regrole::text = 'role3' ORDER BY 1, 2;
+SELECT roleid::regrole::text AS role, member::regrole::text, admin_option, inherit_option, set_option FROM pg_auth_members
+WHERE roleid::regrole::text = 'role3' ORDER BY 1, 2;
 
 DROP ROLE role3, role4, role5;
 
+-- let's grep GRANT remote commands
+RESET citus.log_remote_commands;
+RESET citus.grep_remote_commands;
+
+-- Test that everything works fine for roles that are propagated
+SET citus.enable_ddl_propagation TO on;
+CREATE ROLE role6;
+CREATE ROLE role7;
+CREATE ROLE role9;
+CREATE ROLE role10;
+CREATE ROLE role11;
+CREATE ROLE role12;
+CREATE ROLE role13;
+CREATE ROLE role14;
+CREATE ROLE role17;
+CREATE ROLE role18;
+CREATE ROLE role19;
+CREATE ROLE role20 NOINHERIT;
```

Comment:
Let's just be consistent :) Please fix the following GRANT commands accordingly
```suggestion
CREATE ROLE role15;
CREATE ROLE role16;
CREATE ROLE role17;
CREATE ROLE role18 NOINHERIT;
```